Default Intro

im-adithya opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.

You can learn more about contributing to Sketchify here: https://github.com/s-katte/Sketchify/blob/master/CONTRIBUTING.md

Happy contributing!

-->

## https://github.com/s-katte/Sketchify/issues/42

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

## Description of your PR

Provides default intro in the iFrame

## Screenshots (if applicable)

(Add screenshots of the visual changes in the project)

## Test Plan

I added another iFrame box with a display-block class and on interchanging the class with the previous iFrame, it worked as it previously did.

### Have you read the [Contributing Guidelines](https://github.com/s-katte/React-Code-Pen/blob/master/CONTRIBUTING.md)?

Yes

s-katte committed:
Apply formatting changes

im-adithya committed:
Removed unnecessary comment

Commit:
Added Code_Of_Conduct

im-adithya committed:
Merge pull request #5 from s-katte/master

Update fork

Commit:
Update CODE_OF_CONDUCT.md

As requested, the changes have been made related to he headings of this document.

im-adithya committed:
Added intro for iframe

im-adithya committed:
Merge branch 'master' of https://github.com/im-adithya/Sketchify

s-katte committed:
Update README.md

s-katte committed:
add netlify cd

s-katte committed:
Update README.md

dao-phuong committed:
Apply formatting changes

s-katte committed:
Apply formatting changes

s-katte commented on src/components/App.js:
```diff
@@ -57,8 +79,14 @@ function App() {
         } else {
             document.title = "Sketchify - " + title;
-            localStorage.setItem("codepen-clone-title", title);
+            localStorage.setItem("sketchify-title", title);
```
Please change the spelling here to keep the code consistent.

im-adithya replied:
Sorry I didn't get what you are trying to say...Did you mean capitalizing sketchify?

s-katte replied:
Yes

im-adithya replied:
@s-katte The local storage keys are all set with the "sketchify-title-" prefix; that's why I did this.
Should I still make changes?

s-katte replied:
okay yes, so sorry.

s-katte reviewed (changes requested):
Also, next time please follow the CONTRIBUTING guideline mentioned to avoid extra commits and irregularity in the flow.

s-katte reviewed (approved):
Everything else looks good, merging it.

s-katte reviewed (changes requested):
After some testing, I constantly found that there is an error in the code.
Steps to reproduce error:
- Add a title
- Refresh the page

You can see the error in the console of browser.

Please solve that error, as this occured in only this PR, you can use https://deploy-preview-48--sketchify.netlify.app/

im-adithya committed:
Update fork (#7)

* Added Code_Of_Conduct

* Update CODE_OF_CONDUCT.md

As requested, the changes have been made related to he headings of this document.

* Update README.md

* add netlify cd

* Update README.md

* Apply formatting changes

* Apply formatting changes

Co-authored-by: Shivam <svverma@mitaoe.ac.in>
Co-authored-by: Shivam V. Verma <sv138927@gmail.com>
Co-authored-by: Shubham Katte <39235382+s-katte@users.noreply.github.com>
Co-authored-by: s-katte <shubhamkatte99@gmail.com>
Co-authored-by: dao-phuong <dao-phuong@users.noreply.github.com>
Co-authored-by: s-katte <s-katte@users.noreply.github.com>

im-adithya committed:
Revert "Update fork (#7)"

This reverts commit 128aab925b868e69fb60cd1e8c6706f06fcca1c0.

im-adithya committed:
Merge pull request #8 from im-adithya/revert-7-master

Revert "Update fork"

im-adithya committed:
Fixed title issue

im-adithya commented:
The deploy preview is empty but it's working in my local fork... I don't know what to do now!
Please help @s-katte

s-katte commented:
> The deploy preview is empty but it's working in my local fork... I don't know what to do now!
> Please help @s-katte

Okay, I will merge and try if same issue persists on dev branch.

im-adithya commented:
@s-katte All of my contributions are not counting towards hacktoberfest... It is now showing this as an ineligible repo!

s-katte commented:
> @s-katte All of my contributions are not counting towards hacktoberfest... It is now showing this as an ineligible repo!

were your PRs matured?

im-adithya commented:
No @s-katte all my PRs were about to mature in 2-3 days, nothing got accepted as of now. Also, there's a spelling mistake in repo's Hacktoberfest label. I think it is because of that.

s-katte commented:
> No @s-katte all my PRs were about to mature in 2-3 days, nothing got accepted as of now. Also, there's a spelling mistake in repo's Hacktoberfest label. I think it is because of that.

Thanks for correction, updated that.

im-adithya commented:
It is still showing ineligible. Can you remove and add the hacktoberfest-accepted label now (after updating the repo label) to my other PRs as well?
I have done 3 PRs. None of them got matured now. This is so unfair!

s-katte commented:
> @s-katte It is still showing ineligible. Can you remove and add the hacktoberfest-accepted label now (after updating the repo label) to my other PRs as well?
> I have done 3 PRs. None of them got matured now. This is so unfair!

Yes will do it

im-adithya commented:
It's not updated yet @s-katte. And I don't think it will now. Btw, thanks for your immediate response!
Learned a lot from the project anyway!! 🍻

s-katte commented:
> It's not updated yet @s-katte. And I don't think it will now. Btw, thanks for your immediate response!
> Learned a lot from the project anyway!!

I am glad to have you as a contributor. Looking forward to having you as an active contributor here.
